modules/internal/ChapelArray.chpl

```diff
@@ -1650,6 +1650,11 @@ module ChapelArray {
        return translate(offTup);
      }
 
+     /* Return true if the array has no elements */
```

Comment:
This should say 'domain' instead of 'array'.

Reply:
oh yes, I just copied this comment from array section, I edit it